doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -13,15 +13,20 @@ IMPORTANT: This feature is only accessible by administrator users.
 
 == Exporting Dashbuilder data
 
-You can export all Dashbuilder data from {CENTRAL} as a ZIP file.
+You can export the dashbuilder related data such as datasets, perspectives, and navigation from {CENTRAL} as a ZIP file.
```

Comment:
This is a bit tricky. Previously, pages used to be called perspectives. This is unfortunately not really unified in the business central. You can see that in the export wizard, we have pages but in the export.zip file, we have perspectives. However, I would suggest to use "pages" instead of "perspectives" in this place because in the export wizard, there's "pages" (or put a note here that these terms are interchangeable)

Reply:
Done